.github/workflows/ci.yml

```diff
@@ -66,12 +76,22 @@ jobs:
       - name: Install extra dependencies for building Git on Ubuntu
         if: matrix.targetPlatform == 'ubuntu'
         run: sudo apt-get install libcurl4-openssl-dev libexpat1-dev gettext
-      - name: Build
+      - name: Build (except macOS arm64)
+        if: matrix.targetPlatform != 'macOS' || matrix.arch != 'arm64'
+        shell: bash
+        run: script/build.sh
+        env:
+          TARGET_PLATFORM: ${{ matrix.targetPlatform }}
+          TARGET_ARCH: ${{ matrix.arch }}
+      - name: Build (macOS arm64)
+        if: matrix.targetPlatform == 'macOS' && matrix.arch == 'arm64'
         shell: bash
         run: script/build.sh
         env:
           TARGET_PLATFORM: ${{ matrix.targetPlatform }}
           TARGET_ARCH: ${{ matrix.arch }}
+          # Needed for macOS arm64 until hosted macos-11.0 runners become available
+          SDKROOT: /Library/Developer/CommandLineTools/SDKs/MacOSX11.1.sdk
```

Comment:
I made a specific job for arm64 in order to keep `SDKROOT` used only here, since we're specifically trying to use an older version of Xcode and the SDK for macOS x86_64 for maximum compatibility with older macOS versions (see #350).